Kernel/KString.h

```diff
@@ -52,7 +52,7 @@ class KString {
     }
 
     size_t m_length { 0 };
-    char m_characters[0];
+    char m_characters[0] {};
```

Comment:
Hmm, what does this actually mean? Initializing a zero sized member? 🤔

Reply:
Yea, I believe it does nothing in gcc / clang, but it's enough to quiet static analysis. It's goofy enough that I'll just drop this commit though.